integrated_channels/sap_success_factors/client.py

```diff
@@ -235,3 +239,105 @@ def _call_post_with_session(self, url, payload):
             self._create_session()
         response = self.session.post(url, data=payload)
         return response.status_code, response.text
+
+    def get_inactive_sap_learners(self):
+        """
+        Make a GET request using the session object to a SuccessFactors endpoint for inactive learners.
+
+        Example:
+            sap_search_student_url: "/learning/odatav4/searchStudent/v1/Students?$filter=criteria/isActive eq False"
```

Comment:
We should update this docstring to reflect the actual API call being made (queries for only studentID, returns only studentID).  It's confusing when compared with the `Returns:` bit just below

Reply:
Done